src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -210,7 +210,11 @@ export class IconLabel extends Disposable {
 						};
 						const resolvedTooltip = await tooltip;
 						if (resolvedTooltip) {
-							hoverOptions = { text: resolvedTooltip, target, anchorPosition: AnchorPosition.BELOW };
+							hoverOptions = {
+								text: typeof resolvedTooltip === 'string' ? new MarkdownString().appendText(resolvedTooltip, 'break') : resolvedTooltip,
```

Comment:
Yeah I think this can just be put here:

https://github.com/microsoft/vscode/blob/70227b45b5a96e34228f28d6ba7d9f73b2f7cea7/src/vs/workbench/services/hover/browser/hoverWidget.ts#L85

I think we want this behavior everywhere for regular text passed in. The 2 usages of showHover in the terminal pass in markdown strings so I think this is a safe change to make.